launchdarkly/assets/dashboards/launchdarkly.json

```diff
@@ -3,79 +3,149 @@
+              "background_color": "white",
+              "font_size": "14",
+              "text_align": "left",
+              "vertical_align": "top",
+              "show_tick": false,
+              "tick_pos": "50%",
+              "tick_edge": "left",
+              "has_padding": true
+            },
+            "layout": {
+              "x": 6,
+              "y": 0,
+              "width": 6,
+              "height": 3
+            }
+          }
+        ]
       },
-      "layout": { "x": 5, "y": 2, "width": 7, "height": 7 }
+      "layout": {
+        "x": 0,
+        "y": 0,
+        "width": 12,
+        "height": 4
+      }
     },
     {
       "id": 5250128683109816,
       "definition": {
         "type": "note",
-        "content": "Customize the LaunchDarkly flags widget to show a subset feature flags that are relevant to your team, project, or service.\n\nOptionally, you can utilize a Datadog [template variable](https://docs.datadoghq.com/dashboards/template_variables) to dynamically override the **Launchdarkly environment** setting as you change dashboard views.",
-        "background_color": "gray",
+        "content": "## Create, view, or toggle on or off LaunchDarkly feature flags\n\nControl feature flags directly within this dashboard. Narrow down your search by project or environment.",
+        "background_color": "blue",
         "font_size": "14",
         "text_align": "left",
         "vertical_align": "center",
         "show_tick": true,
         "tick_pos": "50%",
-        "tick_edge": "top",
+        "tick_edge": "bottom",
         "has_padding": true
       },
-      "layout": { "x": 0, "y": 9, "width": 5, "height": 3 }
+      "layout": {
+        "x": 0,
+        "y": 0,
+        "width": 6,
+        "height": 2,
+        "is_column_break": true
+      }
     },
     {
       "id": 1261236294436942,
       "definition": {
         "type": "note",
-        "content": "Enable LaunchDarkly's [Datadog events integration](https://docs.launchdarkly.com/integrations/datadog/events) to send feature flag change events to Datadog. Events are displayed in a list or as markers on your charts. These events help you correlate and understand how changes to your features in LaunchDarkly impact your app and infrastructure metrics.",
-        "background_color": "gray",
+        "content": "## Track Feature Flag Change Events\n\nEvents are displayed in a list or as markers on your charts. These events help you correlate and understand how changes to your features in LaunchDarkly impact your app and infrastructure metrics.\n\nUtilize [template variables](https://docs.datadoghq.com/dashboards/template_variables) to dynamically override the **Launchdarkly environment** setting as you change dashboard views.",
```

Comment:
To confirm, is this correct? `Launchdarkly` or should it be:
`override the **LaunchDarkly environment** setting`